plugins/modules/repository_information.py

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/python
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.0',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: repository_info
+short_description: A module that returns information about GitHub repositories
+description:
+  - "A module that fetches information about repositories that a GitHub user has access to inside an organization."
+options:
+    token:
+        description:
+            - GitHub API token used to retrieve information about repositories a user has access to
```

Comment:
"about repositories a user has access to" -> "about repositories to which a use has access"